# origin: Debian, #192259
# cset: n/a
# inclusion: not submitted
# description: fix size of default_port to allow forced 10mbit operation
# revision date: 2004-09-03

diff -urN kernel-source-2.4.26/drivers/net/via-rhine.c kernel-source-2.4.26-1/drivers/net/via-rhine.c
--- kernel-source-2.4.26/drivers/net/via-rhine.c	2003-11-29 05:26:20.000000000 +1100
+++ kernel-source-2.4.26-1/drivers/net/via-rhine.c	2003-11-29 20:53:36.000000000 +1100
@@ -523,8 +523,8 @@
 	u16 chip_cmd;						/* Current setting for ChipCmd */
 
 	/* These values are keep track of the transceiver/media in use. */
-	unsigned int default_port:4;		/* Last dev->if_port value. */
 	u8 tx_thresh, rx_thresh;
+	unsigned int default_port;		/* Last dev->if_port value. */
 
 	/* MII transceiver section. */
 	unsigned char phys[MAX_MII_CNT];			/* MII device addresses. */
